modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/partials/basic.jsp

```diff
@@ -0,0 +1,41 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<form action="<%= fdsSampleDisplayContext.getPortletURL() %>" method="post" name="fm">
+	<input name="<%= Constants.CMD %>" type="hidden" value="<%= Constants.DELETE %>" />
+	<input name="redirect" type="hidden" value="<%= String.valueOf(fdsSampleDisplayContext.getPortletURL()) %>" />
```

Comment:
This form around the FDS does not look right. Shouldn't all actions be `fetch` requests? :thinking:  In theory, we could make the form work, but wouldn't ajax be a better tool for the job?